src/operators/ops_logic.cpp

```diff
@@ -7,6 +7,7 @@
 #include "../runtime/d_boolean.h"
 #include "../runtime/d_code.h"
 #include "../runtime/d_array.h"
+#include "d_text.h"
```

Comment:
why?

Reply:
Used by `==` and `!=`

Reply:
but it is not used by == and !=, you just added the include

Reply:
`runtime.register_sqfop(binary(3, "==", t_text(), t_text(), "Check if one value is equal to another. Both values need to be of the same type.", equals_any_any));`
`runtime.register_sqfop(binary(3, "!=", t_text(), t_text(), "Returns whether one value is not equal to another.", notequals_any_any));`